### ISSUE TITLE
Getting importing error on Python 3.9.1

### ISSUE THREAD

**Issue by hiddevanesch** [bluesheeptoken/CPT#83]
Getting the following error
```
File "cpt\\cpt.pyx", line 1, in init cpt
ModuleNotFoundError: No module named 'cpt.alphabet'; 'cpt' is not a package
```

**Comment by bluesheeptoken** [comment]
Thanks for raising the issue.

May I ask you:
 - How did you install CPT? (`pip install cpt`?)
 - Which command did you use to run into this error?
 - Which OS does the code run?

**Comment by hiddevanesch** [comment]
> Thanks for raising the issue.
> 
> May I ask you:
> 
> * How did you install CPT? (`pip install cpt`?)
> * Which command did you use to run into this error?
> * Which OS does the code run?

- I tried installing via `pip install cpt`, however, i had to `pip install cython==3.06a` (i think it was 3.06a) to make it work.
- It was `from cpt.cpt import Cpt` as command.
- Code was running with Python 3.9.1 on Windows 10 x64 20H2

**Comment by bluesheeptoken** [comment]
> I tried installing via pip install cpt, however, i had to pip install cython==3.06a (i think it was 3.06a) to make it work.

Mmmh, it feels like something went wrong on the Windows (and macOS but that's not the point here) CIs. No wheels have been published :S. I will try to take time this weekend to fix this.

> Do you have more information about the parameters I can use to specify the model?

If you want more information, you can have a look at this [paper](http://www.philippe-fournier-viger.com/ADMA2013_Compact_Prediction_trees.pdf). Usually, the main increase in accuracy comes from the tuning of the alphabet. (You might want to read this [paper](https://dspace.library.uu.nl/bitstream/handle/1874/327511/thesis-jelte-hoekstra-20150121.pdf) about prediction train journeys)

A quick test might be to try [subseq](https://github.com/bluesheeptoken/subseq), from the same researchers which is a bit more accurate if ordering in the sequence is important.

**Comment by hiddevanesch** [comment]
Thanks! I will check out subseq as well, although this is not currently working for Python 3.9.1 either, but I switched to 3.7.9 now anyway.

**Comment by bluesheeptoken** [comment]
I don't know why I did not take time to work on Python 3.9 at all for subseq

> Subseq gave me an accuracy of 0.518, so that's great!

Great news ! 🎉 

>make sure to change the syntax of .predict,

This is completely right, I should probably harmonize both APIs

> thanks for these implementations of the algorithms!

My pleasure ! If you appreciated the projects, do not hesitate to star / raise more issues as you encounter them. That definitely motivates me for more work 😄

**Comment by bluesheeptoken** [comment]
Hey, sorry for the delay I was a bit busy.

I just released a new version [`1.3.0`](https://pypi.org/project/cpt/1.3.0/) and it should support Python 3.9 :tada:

Can you confirm you can now `pip install` it? I will close the issue meanwhile, do not hesitate to reopen it if that did not fix your issue
